zk/datastream/server/data_stream_server.go

```diff
@@ -264,6 +263,14 @@ func createFullBlockStreamEntriesProto(
 	}
 
 	if l1InfoIndex > 0 {
+		prevIndexBlock, prevIndex, found, err := reader.GetPreviousIndexBlock(blockNum)
+		if err != nil {
+			return nil, err
+		}
+		if found && prevIndex >= l1InfoIndex {
+			log.Warn("1 info index not bigger than previous index", "prevIndex", prevIndex, "prevBlock", prevIndexBlock, "l1InfoIndex", l1InfoIndex, "currentBlock", blockNum)
```

Comment:
@mandrigin 
I think that we only have to decide whether this one should be "Warn" or "panic" and then we can merge it.

Reply:
Let’s keep as Warn, its okay